pixart.js

```diff
@@ -1 +1,36 @@
+var brush = $(".brush");
+var button = document.getElementById("set-color");
+var input = document.getElementById("color-field");
+function changeBrushColor (inputColor) {
```

Comment:
try `$("#set-color")` and `$("color-field")` for lines 3+4 (this is the jQuery way)
`document.getElementById` is the vanilla JS way